[2022-02-08 07:51:39,240] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.2 2022-02-07T00:00:00+00:00 [queued]>
[2022-02-08 07:51:39,437] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.2 2022-02-07T00:00:00+00:00 [queued]>
[2022-02-08 07:51:39,437] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-02-08 07:51:39,438] {taskinstance.py:1088} INFO - Starting attempt 1 of 4
[2022-02-08 07:51:39,438] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-02-08 07:51:39,463] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): 2> on 2022-02-07T00:00:00+00:00
[2022-02-08 07:51:39,472] {standard_task_runner.py:52} INFO - Started process 339 to run task
[2022-02-08 07:51:39,478] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'hello_world', '2', '2022-02-07T00:00:00+00:00', '--job-id', '5', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task.py', '--cfg-path', '/tmp/tmp609pw3wp', '--error-file', '/tmp/tmp6o2npwwx']
[2022-02-08 07:51:39,478] {standard_task_runner.py:77} INFO - Job 5: Subtask 2
[2022-02-08 07:51:39,564] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_world.2 2022-02-07T00:00:00+00:00 [running]> on host 8c4baac7362d
[2022-02-08 07:51:39,820] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=hello_world
AIRFLOW_CTX_TASK_ID=2
AIRFLOW_CTX_EXECUTION_DATE=2022-02-07T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-07T00:00:00+00:00
[2022-02-08 07:51:39,822] {logging_mixin.py:104} INFO - Yeaaahh
[2022-02-08 07:51:39,823] {python.py:151} INFO - Done. Returned value was: None
[2022-02-08 07:51:39,929] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=hello_world, task_id=2, execution_date=20220207T000000, start_date=20220208T075139, end_date=20220208T075139
[2022-02-08 07:51:40,122] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-08 07:51:40,140] {local_task_job.py:151} INFO - Task exited with return code 0
